_layouts/services.html

```diff
@@ -65,90 +68,113 @@ <h1><a href="/revops#expertise">RevOps services</a></h1>
   main .header-overlay {
     position: absolute;
     overflow: hidden;
-    margin-left: 100px;
     background-color: #ecf0f1;
-    opacity: 0.6;
+    opacity: 0.5;
   }
   main .header-image {
     object-fit: cover;
     -webkit-filter: blur(1px);
-    filter: blur(1px);
-    opacity: 0.5;
+    filter: blur(2px);
+    opacity: 0.4;
   }
   main h1 {
     text-align: left;
-    margin: 2em;
-    margin-left: 175px;
-    margin-bottom: 1em;
+    margin: 1.5em;
+    margin-left: 25px;
     margin-right: 25px;
     font-size: 4em;
     font-weight: bold;
     z-index: 1;
   }
-  main section > .image-wrapper {
-    margin-left: 175px;
-    max-width: 650px;
+  main .flex-row {
```

Comment:
CSS selectors seem a bit messy or bloated. Why do you need to be so specific with both `main` and `flex-row` here? Is it not a class that you have created here or is it used elsewhere? In that case I would make a new class to avoid collisions. This is a general comment on the CSS here.